### PR TITLE
Joining Feat/loginTestCypress to branch Dev

### PR DESCRIPTION
**Pull request by sZeuSz**
In this pull request:

Fix login validation fields blank problem;
Add Tests login page using cypress.

### CODE REVIEW

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+/**
```
Comment (calilrenner) [inline cypress/plugins/index.js]: Roseno, what is this comments? Do we need it?

File: cypress/fixtures/example.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```
Comment (calilrenner) [inline cypress/fixtures/example.json]: Do we need it?

**Review by calilrenner — CHANGES_REQUESTED** [review]
Please, can you see if we really need this comments in index.js?

File: cypress/plugins/index.js (hunk 1 above)
Comment (sZeuSz) [reply cypress/plugins/index.js]: No, sorry.

File: cypress/fixtures/example.json (hunk 2 above)
Comment (sZeuSz) [reply cypress/fixtures/example.json]: nop.

**Pull request merged**